src/setcontest.c

```diff
@@ -26,13 +26,65 @@
 #include <stdbool.h>
 #include <string.h>
 
+#include "bandmap.h"
+#include "bands.h"
 #include "focm.h"
 #include "getctydata.h"
 #include "globalvars.h"
 #include "setcontest.h"
 #include "score.h"
 #include "tlf.h"
 
+/* contests where multiplier get determined from comment field can not be
+ shown in bandmap works also for modes with no multiplier at all */
```

Comment:
missing comma is sentence?

Reply:
Did a rewording to (hopefully) make it more clear.

Reply:
Now clear, thanks.